### ISSUE TITLE
Autoadd fails for torrents with certain names

### ISSUE THREAD

**Issue by lewisl9029** [hadouken/hadouken#231]
I recently started trying out Hadouken (coming from Deluge) and am really impressed with it's resource usage so far!

However, Hadouken's autoadd appears to have issues handling certain torrents with non-English filenames. Try putting [this one](https://www.tokyotosho.info/details.php?id=964915) in an autoadd folder to to reproduce. Nothing happens for me when I do this, and in the logs I see the following:

```
(0000016c) 17:09:49.195820: <error> Error when emitting alert 'tick': Error: utf-8 decode failed
```

So it appears to be an issue with the way Hadouken's autoadd handles filename decoding. If I rename the torrent, it would immediately pick up the change and add the torrent. Hadouken also has no problems adding the torrent if I select it manually through the web UI.

I'm using Hadouken 5.2.0 on Windows 10 64-bit. Let me know if you need any more information.

**Comment by Robo210** [comment]
Likely a duplicate of issue #183, which should be available in 5.3.

**Comment by lewisl9029** [comment]
Ah I see. I must have missed that one. Should have done a more thorough search.

I'll close this one for now then. Can't wait to try out 5.3! =)
